spec/features/user_views_food_truck_reviews_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+feature "user views show page and sees reviews", %{
+  As a user
+  I want to view a food truck's reviews
+  So that I can determine where to eat
+} do
+  # Acceptance Criteria
+  # * If a user goes to the show page, they see a list of food truck reviews.
+  scenario "user navigates to a food truck's show page and sees reviews" do
+    f = FactoryGirl.create(:food_truck)
+    FactoryGirl.create(:review, food_truck: f)
```

Comment:
Why not just create the review and let factory girl make the associated food truck?